docs/dev/python/index.md

```diff
@@ -22,3 +23,43 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
+
+<img src="{% link /fig/pyflink.svg %}" alt="PyFlink" class="offset" width="50%" />
+
+PyFlink is a language for building unified batch and streaming workloads.
+This means real-time streaming pipelines, performing exploratory data
+analysis at scale, building machine learning pipelines, and creating ETLs for a data platform.
+If you're already familiar with Python and libraries such as Pandas, then PyFlink makes it simple
+to leverage the full capabilities of the Apache Flink ecosystem.
+
+The PyFlink Table API makes it simple to write powerful relational queries for building reports and
+ETL pipelines.
+At the same time, the PyFlink DataStream API gives developers access to low-level control over
+state and time, unlocking the full power of stream processing.
+
+<div class="row">
+<div class="col-sm-6" markdown="1">
+
+### Try PyFlink
+
+If you’re interested in playing around with Flink, try one of our tutorials:
+
+* [Intro to PyFlink Table API]({% link dev/python/table_api_tutorial.md %})
+* [Intro to PyFlink DataStream API]({% link dev/python/datastream_tutorial.md %})
+
+</div>
+<div class="col-sm-6" markdown="1">
+
+### Explore PyFlink
+
+The reference documentation covers all the details. Some starting points:
+
+* [PyFlink DataStream API]({% link dev/python/table-api-users-guide/index.md %})
```

Comment:
Nit: *PyFlink Table API* is located before the *Python DataStream API* in the *Try PyFlink* section. What about keeping the order consistent?